cafebabel/templates/articles/detail.html

```diff
@@ -54,7 +54,7 @@ <h1>{{ article.title }}</h1>
       <ul>
         <li><span>{{ article.language }}</span></li>
           {% if article.is_translation %}
-            <li class=original-language>Translated from <a href="{{ article.original_article.detail_url }}">{{ article.original_article.language }}</a></li>
+            <li class=original-language><a href="{{ article.original_article.detail_url }}">{{ article.original_article.language }}</a></li>
```

Comment:
If you want to play in CSS with this text you could wrap it with a `span`: `<span>Translated from</span> `
(I suppose that's why you moved it from here to CSS but I may have misunderstood)

Reply:
I thought it was clearer to have _Original language_ than _Translated from_. What do you think?